.github/workflows/test.yml

```diff
@@ -201,7 +201,8 @@ jobs:
           IS_MASTER=${{ github.ref == 'refs/heads/master' }}
           IS_VERIFIED_SCION=${{ github.ref == 'refs/heads/verified-scion-preview-without-selective' }}
           IS_PROPER_TAG=${{ startsWith(github.ref, 'refs/tags/v') }}
-          if [[ $IS_GOBRA_REPO && ($IS_MASTER || $IS_VERIFIED_SCION || $IS_PROPER_TAG) ]]
+          # note that `$IS_GOBRA_REPO == 1` is the same as `!$IS_GOBRA_REPO` because value 1 is interpreted as false in bash
+          if [[ $IS_GOBRA_REPO == 1 && ($IS_MASTER == 1 || $IS_VERIFIED_SCION == 1 || $IS_PROPER_TAG == 1) ]]
```

Comment:
I think this does what we want but it is confusing to read - from reading the condition alone, it looks like the opposite of what we want because everything is negated. I think I would probably change the variable declarations above to
```bash
          # note that these GitHub expressions return the value '1' for 'true', where bash considers "0" to be 'true'.
          # as such, all conditions are negated to produce the expected result in bash
          IS_GOBRA_REPO=${{ github.repository != 'viperproject/gobra' }}
          IS_MASTER=${{ github.ref != 'refs/heads/master' }}
          IS_VERIFIED_SCION=${{ github.ref != 'refs/heads/verified-scion-preview-without-selective' }}
          IS_PROPER_TAG=${{ !startsWith(github.ref, 'refs/tags/v') }}
```
and then I would keep the if-condition as it was before.

Nonetheless, if you prefer it as it is now, fine by me :)

Reply:
Haha that's weird too but maybe `IS_GOBRA_REPO=!${{ github.repository == 'viperproject/gobra' }}` is some middle ground with a comment on the `!`

Reply:
Yes, that is fine ^^